Builds/CMake/RippledCore.cmake

```diff
@@ -1131,5 +1131,6 @@ if (tests)
     # these two seem to produce conflicts in beast teardown template methods
     src/test/rpc/ValidatorRPC_test.cpp
     src/test/rpc/ShardArchiveHandler_test.cpp
+    src/test/ledger/Invariants_test.cpp
```

Comment:
Is there a reason this is included both on line 978 and here?

Reply:
This is a list of files that are built individually even on unity builds. I should double-check that it's still necessary to include here. Or as it says on line 1128,
`# any files that don't play well with unity should be added here`

Reply:
I was mainly questioning whether it still needs to be in its original location as well or if it needs to be moved down here.

Reply:
It would need to be both